src/lightning/pytorch/callbacks/lr_monitor.py

```diff
@@ -193,6 +194,8 @@ def _extract_stats(self, trainer: "pl.Trainer", interval: str) -> Dict[str, floa
             current_stat = self._get_lr_momentum_stat(opt, names)
             latest_stat.update(current_stat)
 
+        trainer.callback_metrics.update({name: torch.tensor(value) for name, value in latest_stat.items()})
```

Comment:
Should this set the current device instead of cpu?

Reply:
Good point. How to get this though? It seems trainer has no default device associated; maybe there is some other way?

Reply:
I think @carmocca meant the following:
```suggestion
        trainer.callback_metrics.update({name: torch.tensor(value, device=trainer.strategy.root_device) for name, value in latest_stat.items()})
```